### PR TITLE
Add infrastructure section and include dask cluster

### PR DESCRIPTION
**Pull request by darribas**

### CODE REVIEW

**Comment by martinfleis** [comment]
I guess we'll have to polish the way of contributing a bit, to push only sources and edits to TOC and Makefile, to have cleaner diff. I'll have a look at it on Monday.

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ build-book:
 	rm -rf book/_build
 	# list folders with notebooks here. Notebooks must be present in _toc.yml.
 	cp -r building_footprints book/code
+	cp -r infrastructure book/
 	jupyter-book build book
 	cp -r book/_build/html docs
 	rm -rf book/code
```
Comment (martinfleis) [inline Makefile:9]: Add `rm -rf book/infrastructure` to delete copied files (to minimise duplication). The idea is that all Jupyter notebooks will be copied into book/code and also deleted on line 9. In the case of markdown sections like this, we should do this manually here.

**Comment by darribas** [comment]
OK I think I've removed changes in `docs` . Somehow this is not building now but I wonder if it's better to merge and figure the build in a different PR?

**Comment by martinfleis** [comment]
Actions are building on merge only. Let's merge this and figure out any issues later.

**Pull request merged**